lib/commands/start.mjs

```diff
@@ -89,11 +95,31 @@ async function uploadArtifacts(apiToken, scenario, workingDir) {
         if (profile.artifactDir) {
             log.debug(`Uploading artifact for profile: ${profile.name}`)
             const artifactDir = path.join(workingDir, profile.artifactDir);
+            assessArtifactDirectoryValidity(artifactDir)
             profile.artifactSha = await uploadArtifactIfNeeded(apiToken, artifactDir);
         }
     }
+    
+    
+
+async function assessArtifactDirectoryValidity(artifact_dir)  {
+    let directorySize= getDirectorySize(artifact_dir);
+    let isSysDir = isSystemDirectory(artifact_dir);
+
+
+    if(directorySize>MAX_DIR_SIZE){
```

Comment:
One space between each side of the operand, and one before the if, like this: ` if (directorySize > MAX_DIR_SIZE){`